installers/osie/main.go

```diff
@@ -4,87 +4,122 @@ import (
 	"context"
 	"strings"
 
+	"github.com/packethost/pkg/env"
+	"github.com/pkg/errors"
 	"github.com/tinkerbell/boots/conf"
+	"github.com/tinkerbell/boots/installers"
 	"github.com/tinkerbell/boots/ipxe"
 	"github.com/tinkerbell/boots/job"
 	"go.opentelemetry.io/otel/trace"
 )
 
-type Installer struct{}
+var (
+	osieURL     = conf.MirrorBaseURL + "/misc/osie"
+	workflowURL = conf.MirrorBaseURL + "/workflow"
 
-// DefaultHandler determines the ipxe boot script to be returned.
-// If the job has an instance with Rescue true, the rescue boot script is returned.
-// Otherwise the installation boot script is returned.
-func (i Installer) DefaultHandler() job.BootScript {
-	return func(ctx context.Context, j job.Job, s ipxe.Script) ipxe.Script {
-		if j.Rescue() {
-			return i.rescue()(ctx, j, s)
+	staticTinkArgs []string
+)
+
+type installer struct{}
+
+func Installer() job.BootScripter {
+	if env.Get("DATA_MODEL_VERSION") != "1" {
```

Comment:
I've lifted this up to here from where it used to be, this env var was being checked for *every* ipxe script that was generated (https://github.com/tinkerbell/boots/blob/main/installers/osie/main.go#L115-L116) instead of just once at boot since they aren't going to change. I'll move this to be done in cmd/boots/main.go and pass it in as args.

Reply:
fixed